operator/internal/pkg/prepare/deploy.go

```diff
@@ -24,12 +25,18 @@ import (
 	"github.com/kyma-project/module-manager/operator/pkg/util"
 )
 
+type InstallCfgType string
+
 const (
-	configReadError = "reading install %s resulted in an error for " + v1alpha1.ManifestKind
+	InstallTargetLocalSecret InstallCfgType = "local-secret"
```

Comment:
```suggestion
	InstallTargetSecret InstallCfgType = "secret"
```
there is no non-local secret strategy, and I would like it to be the same as lifecycle-manager.